gh-actions/common/build-debian/action.yml

```diff
@@ -12,6 +12,10 @@ inputs:
   token:
     required: false
     description: If provided, used for git authentication in the source build
+  lintian:
+    required: false
+    description: Arguments to pass to lintian, if any. Set to false to skip the lintian chek.
+    default: ''
```

Comment:
This should be `true` IMHO (to make it clearer at the reader as I see that by default it runs).